index.html

```diff
@@ -32,31 +32,168 @@ <h1 class="profile">I'm Damilare Glad to see you!</h1>
+                    <div class="lang">
+                        <div class="btn">html</div>
+                        <div class="btn">css</div>
+                        <div class="btn">javascript</div>
+                    </div>
+                    <button type="button" class="bon">See Project</button>
+                </li>        
+           </ul> 
+        </section>
+        <section class="myself">
+            <div class="about">
+                <h1 class="about-myself">About Myself</h1>
+                <p class="myself-content">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p><a href="#" class="connect">LET'S CONNECT</a></p>
+                <ul>
+                    <li>
+                        <a href="#">
+                           <img src="./images/./Twitter-logo.png" alt="Twitter-logo">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                          <img src="./images/Linkedin-logo.png" alt="Linkedin-logo">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                         <img src="./images/M-logo.png" alt="letter">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                          <img src="./images/Github-logo.png" alt="Github-logo">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                          <img src="./images/fingers-logo.png" alt="fingers-logo">
+                        </a>
+                    </li>
+                </ul>
+                <button type="button" class="resume">Get my resume</button>
+            </div>
+            <div class="container">
+                <div>
+                    <div class="lang-skill">
+                        <p class="eng">Languages</p>
+                        <i class="fas fa-chevron-down"></i>
+                    </div>
+                    <div class="dropdown">
+                        <div class="dropdown-menu">
+                            <img src="./images/js-logo.png" alt="javascript-logo">
+                            <p class="java">Javascript</p>
+                        </div>
+                        <div class="dropdown-menu">
+                            <img src="./images/html-logo.png" alt="html-logo">
+                            <p class="java">HTML</p>
+                        </div>
+                        <div class="dropdown-menu">
+                            <img src="./images/css-logo.png" alt="css-logo">
+                            <p class="java">CSS</p>
+                        </div>
+                    </div>
```

Comment:
- [ ] Kindly wrap this in `li` tag. 
- [ ] Also, better to make the dropdown class a `ul` tag and list the dropdown-menu elements inside it. Because you are listing the languages you have used in the project you are showcasing.